src/airflow_tools/providers/filesystem/operators/filesystem.py

```diff
@@ -155,18 +178,9 @@ def execute(self, context: 'Context'):
         filesystem_protocol = FilesystemFactory.get_data_lake_filesystem(
             connection=BaseHook.get_connection(self.filesystem_conn_id),
         )
-        #TODO you can simplify this by using the check_file function directly if the check_specific_filename is not None.
-        # Since the specific file is on the prefix folder, you can just check if the file exists, since if it does, the prefix exists
-        # If the file does not exist, the existance of the prefix do not matter and the check will return False
-        logger.info(f'Checking {self.filesystem_path}')
-        prefix_flag = filesystem_protocol.check_prefix(self.filesystem_path)
-        
-        logger.info(f'Prefix flag: {prefix_flag}')
+
         if self.check_specific_filename:
```

Comment:
Can we make the operator handle this scenario? If the path ends with `/` we can assume that we want to check for a directory, if not for a filename. Do we really need to expose the complexity of choosing the right `check_specific_filename` value depending on the path to the user?